tests/test_workdir.py

```diff
@@ -57,12 +59,53 @@ def test_directory_structure_generated(self, spatial_directory_list):
             assert path.exists()
 
 
-class TestCleanDirectoryTree:
-    """Tests for `clean_directory_tree()`."""
+class TestCleanFiles:
+    """Tests for `clean_directory_tree()` and `clean_cwd_logfiles)."""
 
-    @pytest.mark.parametrize("test_path", [Path("runs"), Path("src")])
-    def test_clean_directory_tree(self, test_path):
-        """Success case: directory tree does not exist after clean."""
-        test_path.mkdir()
-        clean_directory_tree()
-        assert not test_path.exists()
+    @pytest.fixture(autouse=True)
+    def set_internal_cwd(self, monkeypatch):
+        """Sets internal.CWD to pytest's working directory."""
+        monkeypatch.setattr(internal, "CWD", Path.cwd())
+
+    @pytest.fixture()
+    def src_path(self) -> Path:
+        return Path("src")
+
+    @pytest.fixture()
+    def runs_path(self) -> Path:
+        return Path("runs")
+
+    @pytest.fixture(params=["rev_number-0.log", "rev_number-200.log"])
```

Comment:
Is it useful to test names with 2 different numbers?

Reply:
The idea here was to pattern match against multiple possible file names, however it's better to have it as a list